sellotape/sellotape_dj/chat/templates/chat/index.html

```diff
@@ -0,0 +1,24 @@
+<html>
```

Comment:
Is there a specific reason not to make use of the base.html template you guys implemented?